code/src/joystick_value.cpp

```diff
@@ -0,0 +1,37 @@
+#include <joystick_value.hpp>
+
+bool joystick_value_s::operator!=(const joystick_value_s &rhs) {
+    return ((this->x != rhs.x) || (this->y != rhs.y));
+}
+
+joystick_value_s joystick_value_s::operator-(const joystick_value_s &rhs) {
+    return {this->x - rhs.x, this->y - rhs.y};
+}
+
+joystick_value_s joystick_value_s::operator-(const int &rhs) {
+    return {this->x - rhs, this->y - rhs};
+}
+
+joystick_value_s joystick_value_s::operator+(const joystick_value_s &rhs) {
+    return {this->x + rhs.x, this->y + rhs.y};
+}
+
+joystick_value_s joystick_value_s::operator+(const int &rhs) {
+    return {this->x + rhs, this->y + rhs};
+}
+
+bool joystick_value_s::operator>=(const int &rhs) {
+    return (this->x >= rhs || this->y >= rhs);
```

Comment:
are these comparison operators actually correct? 
Is a joystick value `greater than` the other or an int if only one of the two values is bigger?

Reply:
same question for the other operator implementations below this one